src/libstd/f64.rs

```diff
@@ -481,16 +481,16 @@ impl f64 {
     #[stable(feature = "rust1", since = "1.0.0")]
     #[inline]
     #[rustc_deprecated(since = "1.10.0",
-                       reason = "you probably meant `(self - other).abs()`: \
+    reason = "you probably meant `(self - other).abs()`: \
```

Comment:
Why?

Reply:
Sorry for changing the identation 😢 , it might because of my IDE's formatter.